packages/inter-protocol/src/proposals/upgrade-vaults.js

```diff
@@ -151,10 +155,15 @@ export const upgradeVaults = async (
   const vaultFactoryPrivateArgs = kit.privateArgs;
   trace('restarting governor');
 
-  const ecf = await electorateCreatorFacet;
+  const [ecf, newContractGovBundleId] = await Promise.all([
+    electorateCreatorFacet,
+    newContractGovBundleIdP,
+  ]);
+  newContractGovBundleIdErasor.reset();
```

Comment:
IIRC, a `reset()` like this made something more difficult last time. Testing or something. Here's hoping I remember more detail.

Reply:
I think there was 3-way communication that time, and it was conceivable that the second reader would reset before the first reader had read. There are only two parties this time.